Modules/_ssl.c

```diff
@@ -5787,51 +5787,60 @@ sslmodule_add_option(PyObject *m, const char *name, uint64_t value)
 static int
 sslmodule_init_constants(PyObject *m)
 {
-    PyModule_AddStringConstant(m, "_DEFAULT_CIPHERS",
-                               PY_SSL_DEFAULT_CIPHER_STRING);
+    if (PyModule_AddStringConstant(m, "_DEFAULT_CIPHERS",
+                                   PY_SSL_DEFAULT_CIPHER_STRING) < 0) {
+        return -1;
+    }
 
-    PyModule_AddIntConstant(m, "SSL_ERROR_ZERO_RETURN",
+#define _PyModule_ADD_INT_CONST(MODULE, NAME, VALUE) \
```

Comment:
Why not simply `ADD_INT_CONST`? And perhaps the first parameter can be removed, it is always the same.

Reply:
Good question! I've addressed this in my PR description, but here's a visual part:
<img width="696" alt="Снимок экрана 2023-10-24 в 08 57 55" src="https://github.com/python/cpython/assets/4660275/0f2d309a-fc7b-4509-bae2-e92e672dbf29">

`ADD_INT_CONST` does not match the length of the previous aligment. So, you either have to:
- Move the second line as well
- Leave it hanging like this

However, I can remove `m,` part, but I though that it is nice that `_PyModule_ADD_INT_CONST` looks similar `PyModule_AddIntConstant`

Reply:
The `_Py` prefix creates an impression that it is an exported private name. Also, mixing CamelCase with ALL_CAPS looks weird. I suggest to use `ADD_INT_CONST` for consistency with other modules. In many cases it can now be written in a single line.